apps/web/src/services/report.ts

```diff
@@ -42,23 +44,25 @@ export const getReportByEOI = async (filter?: PeriodFilter) => {
   }
 };
 
-export const getTimeRange = (period: PeriodFilter): string => {
-  const from =
-    dayjs(period.from).year() === dayjs(period.to).year()
-      ? dayjs(period.from).format('MMM DD')
-      : dayjs(period.from).format('MMM DD, YYYY');
-
-  return `${from} - ${dayjs(period.to).format('MMM DD, YYYY')}`;
+export const getTimeRange = ({ from, to }: PeriodFilter): string => {
+  const start = dayjs(from).format('MMM DD, YYYY');
+  if (to) {
+    return `${start} ~ ${dayjs(to).format('MMM DD, YYYY')}`;
+  }
+  return `${start} ~ `;
 };
 
-const applyNumberFormat = (sheet: WorkSheet, s: CellAddress, e: CellAddress): void => {
-  for (let r = s.r; r <= e.r; ++r) {
-    for (let c = s.c; c <= e.c; ++c) {
+const applyNumberFormat = (sheet: WorkSheet, rows: any[][]): void => {
```

Comment:
Nit - This function is hard to read, it might be good to add a doc block with a short explaination of it's purpose. 
Also, it would be good to use longer variable names, eg row_number instead of r, and column_number instead of c.

Reply:
I updated the function and fixed the report 1 test case.